### DISCUSSION TITLE
libgcc1 on ubuntu jammy

### DISCUSSION THREAD

**Discussion by adriaanzon** [spatie/browsershot#676] [Q&A]
I'm on Ubuntu 22.04, and when i try running the [recommended Puppeteer installation command](https://spatie.be/docs/browsershot/v2/requirements#content-installing-puppeteer-a-forge-provisioned-server), I get the following error:

```
E: Version '12.1.0-2ubuntu1~22.04' for 'libgcc1' was not found
```

It seems like this package does not exist anymore on jammy, or at least not under this name. Does anyone know which package to install instead?

**Comment by sbaker1984** [comment]
https://github.com/spatie/browsershot/discussions/663 

i think the answer to your question is here
